JS/pop-up.js

```diff
@@ -0,0 +1,42 @@
+const containerBox = document.querySelector('.work-section__box');
```

Comment:
- [ ] As requird :point_down: 

![image](https://user-images.githubusercontent.com/57421823/151015393-2368bdbc-0f28-46f4-9357-b17c3811e52d.png)

As seen above, all the projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for projects list should be removed and instead it should be added via JavaScript.

This means you can use the `projects` array and logic that you already have, to populate the projects section of the main page entirely with javascript!

If you are struggling or don't know how to do so, here is an [example](https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95)

Reply:
Thank you for pointing that out. took me a while to understand what was need of me but I finally fixed it. Let me know if everything is working as it was required.